app/views/users/show.html.erb

```diff
@@ -1,22 +1,25 @@
 <div id="review-list">
   <header>
     <h2><%= @user.username %></h2>
+    <div id="sort-bar">
+      <a href="<%= user_path(@user) + "?sort=date-desc" %>">Sort by Newest</a>
+      <a href="<%= user_path(@user) + "?sort=date-asc" %>">Sort by Oldest</a>
+    </div>
   </header>
-  <% @user.reviews.each do |review| %>
+  <% @user.sorted_reviews(params[:sort]).each do |review| %>
```

Comment:
Hm... I don't think we should be using params in the view. Maybe pass in an ivar from the controller: `@sort_order=params[:sort]` and then in the view `@user.sorted_reviews(@sort_order)`. Would that work?

Reply:
Refactoring on branch to match MVC conventions.